src/components/Sale/SelectedProducts.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-container v-if="props.products.value?.length > 0">
+    <div class="mx-auto text-center" style="font-size: 16px">
+      Currently selected products:
+    </div>
+    <Product-price-discount-row
+      v-for="(product, i) in props.products.value"
+      :key="product"
+      :product="props.products.value[i]"
+    >
+    <SaleToolTipProduct :sale-data="props.products.value" :product="product" />
+    </Product-price-discount-row>
```

Comment:
Why is this in kebab-case? I see other components are in PascalCase

Reply:
updated